stock_barcodes/models/stock_barcodes_option.py

```diff
@@ -97,6 +97,12 @@ class StockBarcodesOptionGroup(models.Model):
     use_location_dest_putaway = fields.Boolean(
         string="Use location dest. putaway",
     )
+    location_field_to_order = fields.Selection(
```

Comment:
Although it can be considered correct, it's better to use `sort` to differentiate this from order as document (sales order, purchase order...). Is it too much to change it?

```suggestion
    location_field_to_sort = fields.Selection(
```